Modules/CalcOffence.lua

```diff
@@ -2167,6 +2179,9 @@ function calcs.offence(env, actor, activeSkill)
 		if skillData.dpsMultiplier then
 			t_insert(breakdown.TotalDPS, s_format("x %g ^8(DPS multiplier for this skill)", skillData.dpsMultiplier))
 		end
+		if output.CullMultiplier and output.CullMultiplier > 1.0 then
```

Comment:
```suggestion
		if output.CullMultiplier and output.CullMultiplier > 1 then
```
No need to use floating-point numbers here. In Lua < 5.3, every number is a double.